README.md

```diff
@@ -6,6 +6,56 @@ The mobile-user-contact microservice provides an API that allows mobile app user
 
 Currently this is implemented using Deskpro (via the `hmrc-deskpro` microservice).
 
-### License
+## License
 
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html")
+
+### Standard Responses
+
+We use standard HTTP status codes to show whether an API request succeeded or not. They're usually:
+
+|Status Range| Meaning |
+|------------|--------|
+|in the 200 to 299 range| if it succeeded; including code 202 if it was accepted by an API that needs to wait for further action|
+|in the 400 to 499 range| if it didn't succeed because of a client error by your application|
+|in the 500 to 599 range| if it didn't succeed because of an error on our server|
+
+###  Submit Feedback
+>This resource is user-restricted - it requires an Authorization header containing an OAuth 2.0 Bearer Token
```

Comment:
Do we need to mention what scope the authorization needs?

Reply:
I don't think we so; the canonical source for that level of detail should be the developer hub itself. This is more of a courtesy for the app developers to reference.